docs/book/src/developer/architecture/controllers/support-multiple-instances.md

```diff
@@ -0,0 +1,41 @@
+# Support running multiple instances of the same provider
+
+Up until v1alpha3, the need of supporting multiple credentials was addressed by running multiple
```

Comment:
I feel like this sentence may be missing something when read by someone new to CAPI. 
That is, the reader may ask _What is the need for supporting multiple credentials?_ or _What does multiple credentials have to do with multi-tenancy of providers?_

But I'm unsure how to word this myself. My thoughts lead me to - multiple credentials required to define workload clusters within separate IaaS accounts.

Reply:
I noticed that in the Glossary section we have 

> Multi tenancy in Cluster API defines the capability of an infrastructure provider to manage different credentials, each one of them corresponding to an infrastructure tenant.

Maybe we just link to the Multi tenancy glossary entry here.